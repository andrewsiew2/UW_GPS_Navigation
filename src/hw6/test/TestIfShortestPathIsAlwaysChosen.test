# Tests if the path chosen is always the lexicographically least path

LoadGraph tester testing.tsv

FindPath tester THANOS SPIDERMAN