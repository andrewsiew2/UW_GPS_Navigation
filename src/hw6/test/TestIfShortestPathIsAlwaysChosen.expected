# Tests if the path chosen is always the lexicographically least path

loaded graph tester

path from THANOS to SPIDERMAN:
THANOS to GROOT via 1
GROOT to SPIDERMAN via 2